### DISCUSSION TITLE
Do you see duplicate lists in Ticktick?

### DISCUSSION THREAD

**Discussion by thesamim** [thesamim/TickTickSync#51] [Polls]
I am assuming that I have a number of duplicate lists (Projects) in TickTick because of the weird testing I'm doing.

Is anyone else seeing duplicate lists?

**Comment by Dnartskel** [comment]
Yes, a lotttt

**Reply by Dnarts** [reply]
+

**Reply by Dnarts** [reply]
A. Wait. No. Only duplicates tasks in my ticktick. Then I think obsidian sync them backwards again, and again(maybe)

**Reply by thesamim** [reply]
ok, well duplicate tasks I've found the root cause for and I'm fixing. Or working on it. Not sure if I'm going to be able to automagically get rid of dupes, but will look into it.

**Reply by thesamim** [reply]
@Dnarts: Please update to 1.0.18 and let me know results. Thanks.
